javascript/javascript3/week3/screenshot.js

```diff
@@ -0,0 +1,103 @@
+console.log("Screenshot API");
+
+// STEP 1: FETCH DATA FROM API
+//Fetch Data from API
+
+// Select the button, when the button is clicked, get the input value.
+const searchButton = document.querySelector("#generate-screenshot");
+
+// Display Screenshot
+const displayScreenshot = document.querySelector("#image");
+
+// Save Screenshot
+const saveScreenshotButton = document.querySelector(".saveScreenshot");
+
+const getScreenshot = async (event) => {
+    // Search Input
+    const searchInput = document.querySelector("#url").value
+
+    if (!searchInput) {
+        alert("Please enter a url");
+        return;
+    }
+
+    const HOST_URL = "https://website-screenshot6.p.rapidapi.com/screenshot"
+    const XRapidAPIKey = "f797f72385msh65df0fcdf98a7dep1ff9b3jsn9efc4dfb6fac";
+    const XRapidAPIHost = "website-screenshot6.p.rapidapi.com";
+    const url = searchInput;     //"https://rapidapi.com/marketplace" //"https://www.hackyourfuture.dk/" //"https://rapidapi.com/marketplace";
+    const width = 1920;
+    const height = 1080;
+
+    const options = {
+        method: 'GET',
+        headers: {
+            'X-RapidAPI-Key': 'f797f72385msh65df0fcdf98a7dep1ff9b3jsn9efc4dfb6fac',
```

Comment:
You already (sadly) have these as variables, why retype them?

Reply:
Made corrections in resubmission